src/components/Social/SocialMedia.tsx

```
@@ -1,41 +1,24 @@
 "use client";
 
+import { config } from "@/src/common/appconfig";
 import React from "react";
```

ESLint

[error] Expected "react" (external) to come before "@/src/common/appconfig" (internal).

(perfectionist/sort-imports)